src/vivarium/examples/disease_model/__init__.py

```diff
@@ -4,3 +4,15 @@
 from .disease import DiseaseModel, DiseaseState, DiseaseTransition, SIS_DiseaseModel
 from .risk import Risk, DirectEffect
 from .intervention import MagicWandIntervention
+
+
+def get_model_specification_path():
```

Comment:
Both of these are convenience methods for tutorials.  Otherwise users won't be able to access the files in their installation.